src/data/skill-desc.csv

```diff
@@ -421,9 +421,9 @@
 "レース終盤に前方が詰まると\nお告げ次第で道をわずかに切り開く","<size=22>Slightly easier to find an opening if blocked at the front in Final Leg</size>"
 "レース終盤で3番手の時に負けそうに\nなると速度がわずかに上がる","<size=22>When you're in 3rd place at the end of the race and about to lose, your speed slightly increases</size>"
 "レースを冷静に運ぶことができると残り\n200mで実力発揮、速度が少し上がる","<size=22>After proceeding through the race calmly, your speed increases a little during the last 200m</size>"
-"中盤で2回スキルを使うと\n持久力がわずかに回復する",""
+"中盤で2回スキルを使うと\n持久力がわずかに回復する","Restore endurance slightly if you activate two skills in the middle part of the race"
 "最終直線で前の方で追いすがっていると\n速度がわずかに上がる","<size=22>Slightly increase speed in the last straight when chasing someone close to the lead</size>"
-"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる",""
+"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In the second half of the race, increase speed slightly when you recover endurance with your skills while being in front</size>"
```

Comment:
```suggestion
"スキルで持久力を回復すると\nレース後半で前のほうにいる時に\nわずかに速度を上げる","<size=22>In second half of the race, increase speed slightly when you recover endurance with your skills while being in front</size>"
```

Reply:
See below suggestion by me, similar to your other suggestion above